### ISSUE TITLE
Create ember-try auth0 scenarios

### ISSUE THREAD

**Issue by seawatts** [seawatts/ember-simple-auth-auth0#39]

**Comment by cs3b** [comment]
Just my thoughts:
is it feasible to switch from bower to npm packages and use greenkeeper to create pull request, and have build on travis for each version bump?

**Comment by seawatts** [comment]
Yeah I've been thinking about that. I just read this https://simplabs.com/blog/2017/02/13/npm-libs-in-ember-cli.html
